presentation/src/main/java/com/binhth/photocollection/ui/screen/core/BaseListViewModel.kt

```diff
@@ -27,18 +27,23 @@ abstract class BaseListViewModel<T> : BaseViewModel() {
         requestData(1)
     }
 
-    fun loadMore(page: Int) {
+    fun loadMore() {
         isLoadMore.value = true
-        requestData(page)
+        requestData(currentPage.value?.plus(1) ?: 1)
     }
 
 
-    override fun onLoadSuccess(page: Int) {
-        super.onLoadSuccess(page)
+    open fun onLoadSuccess(page: Int, items: List<T>) {
+        super.onLoadSuccess()
         isLoadMore.value = false
         isRefresh.value = false
         isLoadFailed.value = false
         currentPage.value = page
+
+        //Reassign value of list with temp list from new value
+        val tempList = listItem.value ?: arrayListOf()
+        tempList.addAll(items)
+        listItem.value = tempList
```

Comment:
A làm thế này thì listItem không được add thêm vào mà hoàn toàn mới, bên view như em vừa cmt lúc này cũng thế thì loadmore có chạy không nhỉ? a check lại xem

Reply:
listItem lúc này nhận thêm giá trị rồi em. vì a có gán lại tempList với giá trị cũ của listItem rồi mới add thêm item.